components/Marker/Marker.tsx

```diff
@@ -27,25 +35,28 @@ const Marker = ({
     >
       <Popup>
         <div>
-          <div className={styles.text}>
-            <h1>
-              {city},<br />
-              {country}
-            </h1>
-            {date && (
-              <i>
-                {getFullDateString(date)} ({getRelativeTimeString(date)})
-              </i>
-            )}
-            <br />
-            <span>{name}</span>
+          <div className={styles.background}>
+            <div className={styles.text}>
+              <h1 className={styles.title}>
+                {city},<br />
```

Comment:
Maybe we don't need the <br/> anymore? What do you think?

Reply:
You mean like:

`{city}, {country}`

![image](https://user-images.githubusercontent.com/80540164/193418047-4f7df8bb-8c8a-44a0-9b0d-17eb3f1cecdf.png)

?

Reply:
I think I prefer it. What do you like more?

Reply:
I prefer it on most cases, but sometimes it looks like this:

![image](https://user-images.githubusercontent.com/80540164/193418160-45a0d126-e1a2-4b0e-95fa-a48ac94f8269.png)

Or this one: (now with the left spacing the same as the right)

![image](https://user-images.githubusercontent.com/80540164/193418207-0b9fd11d-f7e0-411a-9b48-8a8f659d8bc1.png)

I don't think it necessarily looks bad, just different. I think it's worth it, what do you think?

Reply:
I still prefer this way :+1:

Reply:
It's done then :) :rocket: